samples/2-ConnectToGraph.ps1

```diff
@@ -6,6 +6,9 @@ Connect-Graph
 # Try to Get-User
 Get-MgUser
 
+# Search for delegated permissions related to sites
```

Comment:
@peombwa goal here is to show you can use the new command to find permissions, and also to get to the permissions reference help.